WordPress/Classes/ViewRelated/Views/WPRichText/WPRichContentView.swift

```diff
@@ -155,6 +166,27 @@ class WPRichContentView: UITextView {
         }
     }
 
+    private func setupTouchDetection() {
+          for gesture in gestureRecognizers ?? [] {
+              gesture.require(toFail: linkTapGestureRecognizer)
+          }
+          addGestureRecognizer(linkTapGestureRecognizer)
+      }
+
+      @objc func tapRecognized(_ recognizer: UIGestureRecognizer) {
+          let point = recognizer.location(in: self)
+          let characterIndex = self.layoutManager.characterIndex(for: point, in: self.textContainer, fractionOfDistanceBetweenInsertionPoints: nil)
+          // handle tap on link
+          if let linkAttribute = self.attributedText?.attribute(.link, at: characterIndex, effectiveRange: nil),
+              let url = linkAttribute as? URL,
+              let richDelegate = delegate as? WPRichContentViewDelegate {
+              richDelegate.interactWith?(URL: url)
+          // handle tap on attachement
+          } else if let attachmentAttribute = self.attributedText?.attribute(.attachment, at: characterIndex, effectiveRange: nil),
+              let attachment = attachmentAttribute as? WPTextAttachment {
```

Comment:
This doesn't seem to be getting called. When you tap on an image in a reader post, I'm seeing a `ReaderPlaceholderAttachment` instead of `WPTextAttachment`. Opening the image is working because the attachment manager is creating a `WPRichTextImage` which is a `UIControl` and acts as a button.

Unless I missed other details, we can get rid of this case.

Reply:
Weird, it's getting called for me and if I remove this logic, tap on an image doesn't do anything.

Reply:
I was testing on the iOS 13.1 simulator

Reply:
I'm on 13.1 device